rest/model/monitor/job_test.go

```diff
@@ -63,13 +63,21 @@ func TestUnmarshalJobs(t *testing.T) {
 	if conf["host"] != "1.2.3.4" {
 		t.Error("Wrong host")
 	}
-	status := j.Status["global"]
-	if status.Since != 1389407609 {
+
+	if j.Status["global"].Since != 1389407609 {
 		t.Error("since has unexpected value")
 	}
-	if status.Status != "up" {
+	if j.Status["global"].Status != "up" {
 		t.Error("Status is not up")
 	}
+
+	if j.Status["sjc"].Since != 1389404014 {
```

Comment:
The conditional could have been "lga" as well, its purpose is to test that each region has its own `monitor.Status` instance.

This test case would fail before, when `Job.Status` was defined as `Status map[string]Status` instead of `Status map[string]*Status`